JavaScript/CodingConventions.md

```diff
@@ -262,7 +262,7 @@ Style rules help us write easy to read, well documented, and consistent code.
 
 ### White Space Guidelines
 
-Our whitespace guidelines are based on JSLint. There are also some rules listed below that are in addition JSLint.
+These guidelines also apply to JSON, where applicable. Our whitespace guidelines are based on JSLint. There are also some rules listed below that are in addition JSLint.
```

Comment:
I think it would read better if this sentence was at the end of the paragraph. Other than that this LGTM.